.tx/config

```diff
@@ -36,38 +36,255 @@ source_lang  = en
+source_file  = config/locales/views/home/contact.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-home-datasets-yml]
+file_filter  = config/locales/<lang>/views/home/datasets.yml
+source_file  = config/locales/views/home/datasets.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-home-education_workshops-yml]
+file_filter  = config/locales/<lang>/views/home/education_workshops.yml
+source_file  = config/locales/views/home/education_workshops.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-home-energy_audits-yml]
+file_filter  = config/locales/<lang>/views/home/energy_audits.yml
+source_file  = config/locales/views/home/energy_audits.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-home-enrol-yml]
+file_filter  = config/locales/<lang>/views/home/enrol.yml
+source_file  = config/locales/views/home/enrol.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-home-enrol_our_local_authority-yml]
+file_filter  = config/locales/<lang>/views/home/enrol_our_local_authority.yml
+source_file  = config/locales/views/home/enrol_our_local_authority.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-home-enrol_our_multi_academy_trust-yml]
+file_filter  = config/locales/<lang>/views/home/enrol_our_multi_academy_trust.yml
+source_file  = config/locales/views/home/enrol_our_multi_academy_trust.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-home-enrol_our_school-yml]
+file_filter  = config/locales/<lang>/views/home/enrol_our_school.yml
+source_file  = config/locales/views/home/enrol_our_school.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-home-for_local_authorities-yml]
+file_filter  = config/locales/<lang>/views/home/for_local_authorities.yml
+source_file  = config/locales/views/home/for_local_authorities.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-home-for_multi_academy_trusts-yml]
+file_filter  = config/locales/<lang>/views/home/for_multi_academy_trusts.yml
+source_file  = config/locales/views/home/for_multi_academy_trusts.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:views-home-for_schools-yml]
+file_filter  = config/locales/<lang>/views/home/for_schools.yml
+source_file  = config/locales/views/home/for_schools.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
 [o:energy-sparks:p:energy-sparks:r:views-home-home-yml]
 file_filter  = config/locales/<lang>/views/home/home.yml
 source_file  = config/locales/views/home/home.yml
 source_lang  = en
 type         = YML
 minimum_perc = 0
 
```

Comment:
do we also need a config for `config/locales/views/home/privacy_and_cookie_policy.yml` here?

Reply:
Yes, this was missing. Dodgy regex in the rake task.